### PR TITLE
Make Fancy Scrollbars Fancier

### LINKED ISSUE #11303: Custom Scrollbars
Since the maintainer of `react-custom-scrollbars` has stated they will not resolve the mac osx mouse settings issue, we will have to solve it ourselves.

Option 1:
Fork `react-custom-scrollbars` and see if we can fix the issue ourselves

Option 2:
Build a custom scrollbar library from scratch. Ideally we are not re-inventing the wheel

Option 3:
If previous options seem unsatisfactory we can rebuild native scrolling functionality. Ideally things don't come to this point

### PR DESCRIPTION
**Pull request by alexpaxton**
Closes #11303 

- Fork `react-custom-scrollbars` and tailor to our needs
- Fix issue of intermittent disappearance of scrollbar on mac laptops
- Appearance is less customizable but hardcoded to match the InfluxDB aesthetic
- Not related to this issue, but I removed `yarn.lock` since it is somehow a tracked file

#### Checklist

- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: ui/package-lock.json [hunk 1]
```diff
@@ -836,6 +836,16 @@
         "axios": "^0.18.0"
       }
     },
+    "@influxdata/react-custom-scrollbars": {
+      "version": "4.3.8",
+      "resolved": "https://registry.npmjs.org/@influxdata/react-custom-scrollbars/-/react-custom-scrollbars-4.3.8.tgz",
```
Comment (OfTheDelmer) [inline ui/package-lock.json]: 💯

File: ui/src/shared/components/fancy_scrollbar/FancyScrollbar.scss [hunk 2]
```diff
@@ -1,87 +0,0 @@
-/*
-  Fancy Scrollbars
-  ------------------------------------------------------
-*/
-
-$scrollbar-track-size: 12px;
```
Comment (OfTheDelmer) [inline ui/src/shared/components/fancy_scrollbar/FancyScrollbar.scss]: ✂️

File: ui/src/style/chronograf.scss [hunk 3]
```diff
@@ -42,3 +41,4 @@
 @import 'src/logs/components/logs_table/LogsTable';
 @import 'src/logs/components/expandable_message/ExpandableMessage';
 @import 'src/logs/components/logs_message/LogsMessage';
+@import '../../node_modules/@influxdata/react-custom-scrollbars/dist/styles.css';
```
Comment (OfTheDelmer) [inline ui/src/style/chronograf.scss]: Do we want to move node_module styles to a separate section?

File: ui/yarn.lock [hunk 4]
```diff
@@ -1,9316 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (OfTheDelmer) [inline ui/yarn.lock]: NICE!

Comment (alexpaxton) [reply ui/yarn.lock]: Boooom!

File: ui/src/style/chronograf.scss (hunk 3 above)
Comment (alexpaxton) [reply ui/src/style/chronograf.scss]: Ahh that's a good call

**Pull request merged**